assessments/0003-krator.md

```diff
@@ -0,0 +1,181 @@
+    - **Tasks:** I recommend a section with step-by-step instructions on how to accomplish the most common tasks in Krator. For example, if you have five most common tasks, you could have a document for each. Suggestions include:
+        - **Using built-in operators**
+        - **Creating your own operators with krator**
+        - Are there other common tasks? If so, they should go here.
+    - **Best practices**
+    - **Troubleshooting**
+        - **Troubleshooting** 
+        - **Error Reference** A table with explanations and resolution steps would suffice
+    - **Contributing to krator**
+        - **Cutting releases** (This is existing documentation)
+        - **Contributing:** Include information on submitting issues and instructions with links on contributing to the code base and to the documentation.
+    - **API Documentation:** If you choose to bring this into the main documentation repo, the copy will need to be ported. Is the content available in markdown already? If not, it will need to be converted. 
+
+## Contributor documentation 
+
+| Criteria | 1 = (Is not present or requires significant work) | 2 | 3 = (is present, but needs work) | 4 | 5 = (is executed extremely well or no improvement required) |
+| ---                       | --- | --- | --- | --- | --- |
+| Communication methods documented  |  ✅  |     |     |     |     |
+| Beginner friendly issue backlog   |  ✅  |     |     |     |     |
+| “New contributor” getting started content  |  ✅  |     |     |     |     |
+| Project governance documentation |  ✅  |     |     |     |     |
+
+### Comments and recommendations
+
+Communication methods will need to be determined and added to [CONTRIBUTING.md](https://github.com/krator-rs/krator/blob/main/CONTRIBUTING.md).
+
+While there is an issue backlog, there aren't any issues with a `good first issue` label. As an invitation to new contributors, consider filing issues that are easy to remedy, labeling them accordingly, posting them on social media, and being available to help contributors through the process.  
+
+There is a brief mention of the PR review process in [CONTRIBUTING.md](https://github.com/krator-rs/krator/blob/main/CONTRIBUTING.md), however, giving more detail about the process, such as rounds of review, time frame, a link to [CODEOWNERS](https://github.com/krator-rs/krator/blob/main/CODEOWNERS), tests that have to pass, etc. help manage contributor expectations. This, along with linking to the Code of Conduct as you currently do, establishes a context for new contributors. 
+
+
+
+## Website
+
+| Criteria | 1 = (Is not present or requires significant work) | 2 | 3 = (is present, but needs work) | 4 | 5 = (is executed extremely well or no improvement required) |
+| ---                       | --- | --- | --- | --- | --- |
+| Branding and design  |  ✅  |     |     |     |     |
+| Case studies/social proof |  ✅  |     |     |    |     |
+| Maintenance planning |  ✅  |     |     |     |     |
+
+### Comments
```

Comment:
```suggestion
### Comments

Note: as Krator does not have a website yet, we did not expect any of these to be done!
```

Reply:
Also: I added this just to soften the blow for the Krator team 😅. It might be a bit difficult for a small team with no resources to receive "bad marks" so to speak for something they don't already have, hence this.